cdap-docs/developer-manual/source/building-blocks/spark-programs.rst

```diff
@@ -73,14 +73,6 @@ When a Spark program is configured, the resource requirements for both the Spark
 processes and the Spark executor processes can be set, both in terms of the amount of
 memory (in megabytes) and the number of virtual cores assigned.
 
-For example, in the :ref:`Spark Page Rank <examples-spark-page-rank>` example, in the configuration of
-the ``PageRankSpark``, the amount of memory is specified:
-
-.. literalinclude:: /../../../cdap-examples/SparkPageRank/src/main/java/io/cdap/cdap/examples/sparkpagerank/SparkPageRankApp.java
-   :language: java
```

Comment:
similar to above, can we have literal code to show the properties that need to be set?